src/topology/algebra/module.lean

```diff
@@ -782,17 +803,29 @@ instance : semimodule S (M →L[R] M₂) :=
   add_smul  := λ _ _ _, ext $ λ _, add_smul _ _ _,
   smul_add  := λ _ _ _, ext $ λ _, smul_add _ _ _ }
 
-variables (S) [has_continuous_add M₃]
-
-/-- `continuous_linear_map.prod` as a `linear_equiv`. -/
-def prodₗ : ((M →L[R] M₂) × (M →L[R] M₃)) ≃ₗ[S] (M →L[R] M₂ × M₃) :=
+/-- `continuous_linear_map.prod` as an `equiv`. -/
+@[simps apply] def prod_equiv : ((M →L[R] M₂) × (M →L[R] M₃)) ≃ (M →L[R] M₂ × M₃) :=
```

Comment:
What's the reason for having both the linear and basic equiv?

Reply:
The basic equiv doesn't need `[has_continuous_add M₃]` (though in all reasonable cases we have this assumption anyway).